docs/jest-integration.md

```diff
@@ -283,6 +284,28 @@ Time:        0.335 s, estimated 1 s
 Ran all test suites.
 ```
 
+### Coverage report generation
+
+To generate a coverage report, run jest with the `--coverage` flag:
+
+```shell
+» npx jest --coverage
+```
+
+Files matched by the flags `--include` and `--custom_hooks`, and not matched by
+the flag `--exclude` will be included in the coverage report. These can be
+specified in the `package.json` or `.jazzerjsrc.json` config files as described
+above.
+
+By default, the coverage reports can be found in the `./coverage` directory.
+This default directory can be changed by setting the flag
+`--coverageDirectory=<another coverage directory>`.
+
+The desired report format can be set by the flag `--coverageReports`, which by
+default is set to `--coverageReports clover json lcov text`. See
+[here](https://github.com/istanbuljs/istanbuljs/tree/master/packages/istanbul-reports/lib)
+for a list of supported coverage reporters.
```

Comment:
Could you remove this duplicate text and add a link to the corresponding section in `fuzz-targets/md`

Reply:
Removed and added link.